### DISCUSSION TITLE
export 504 Gateway Time-out

### DISCUSSION THREAD

**Discussion by Pixflowave** [EcrituresNumeriques/stylo#1164] [Q&A] [answered]
Bonsoir,
je n'ai pas réussi à faire d'export .pdf ou .odt . Je n'ai réussi qu'a obtenir une version html incompléte ...

**Comment by thom4parisot** [comment]
Sur un article ou un corpus ?

**Reply by Pixflowave** [reply]
Un article

**Reply by thom4parisot** [reply, marked as answer]
Je confirme le problème ; j'ai ouvert l'issue #1165 pour détricoter les symptômes.
